metadata-ingestion/src/datahub/ingestion/source/sql/bigquery.py

```diff
@@ -305,21 +301,24 @@ def get_db_name(self, inspector: Inspector = None) -> str:
             return self._get_project_id(inspector)
 
     def _compute_big_query_lineage(self) -> None:
-        if self.config.include_table_lineage:
-            lineage_client_project_id = self._get_lineage_client_project_id()
-            if self.config.use_exported_bigquery_audit_metadata:
-                self._compute_bigquery_lineage_via_exported_bigquery_audit_metadata(
-                    lineage_client_project_id
-                )
-            else:
-                self._compute_bigquery_lineage_via_gcp_logging(
-                    lineage_client_project_id
-                )
+        if not self.config.include_table_lineage:
+            return
 
-            if self.lineage_metadata is not None:
-                logger.info(
-                    f"Built lineage map containing {len(self.lineage_metadata)} entries."
-                )
+        lineage_client_project_id = self._get_lineage_client_project_id()
+        if self.config.use_exported_bigquery_audit_metadata:
+            self._compute_bigquery_lineage_via_exported_bigquery_audit_metadata(
+                lineage_client_project_id
+            )
+        else:
+            self._compute_bigquery_lineage_via_gcp_logging(lineage_client_project_id)
+
+        if self.lineage_metadata is None:
```

Comment:
Instead of this, ensure that self.lineage_metadata is an empty dict. That way, we will be able to log&report cases where there is no lineage_metadata as well.

Reply:
In the next line we have which reports the length of dictionary keys
```
self.report.lineage_metadata_entries = len(self.lineage_metadata)
```

Reply:
It would report 0. I don't see any issues with the change. In fact, it would indicate that the lineage processing code has run but found 0 entries, which is good from troubleshooting perspective.